airbyte-integrations/connectors/source-open-exchange-rates/setup.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from setuptools import find_packages, setup
+
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk~=0.2",
+]
+
+TEST_REQUIREMENTS = [
+    "pytest~=6.1",
+    "pytest-mock~=3.6.1",
+    "pendulum==2.1.2",
```

Comment:
This should be in `MAIN_REQUIREMENTS` instead of `TEST_REQUIREMENTS`

Reply:
Sorry if I was unclear, I meant only `pendulum`, not the others `pytest`/`pytest-mock`